packages/flutter_tools/lib/src/device.dart

```diff
@@ -334,6 +434,56 @@ abstract class DeviceDiscovery {
   ///
   /// For example, 'windows' or 'linux'.
   List<String> get wellKnownIds;
+
+  bool exactlyMatchesDeviceId(String deviceId, Device device) {
```

Comment:
are device IDs not case sensitive? and is this true across platforms?

Reply:
i suppose if this is the comparison we were making before then we should keep doing it. it's not obvious from the diff here if we were doing a similar comparison previously.

Reply:
Yeah sorry about the diff. It is the same as before.